lib/kubernetes-deploy/errors.rb

```diff
@@ -0,0 +1,3 @@
+module KubernetesDeploy
+  class FatalDeploymentError < StandardError; end
```

Comment:
I was intending to move it, but apparently it got lost in the rebase, and now I don't remember why it was necessary. I'll put it back.